ledger-core/runner/execution/rpc.go

```diff
@@ -101,17 +101,17 @@ func (e CallConstructor) ParentRequestReference() reference.Global {
 	return e.parentRequestReference
 }
 
-func (e CallConstructor) ConstructVCallRequest(execution Context) *payload.VCallRequest {
-	return &payload.VCallRequest{
-		CallType:            payload.CallTypeConstructor,
-		CallFlags:           payload.BuildCallFlags(execution.Isolation.Interference, execution.Isolation.State),
-		Caller:              e.parentObjectReference,
-		Callee:              e.class,
+func (e CallConstructor) ConstructVCallRequest(execution Context) *rms.VCallRequest {
+	return &rms.VCallRequest{
+		CallType:            rms.CallTypeConstructor,
+		CallFlags:           rms.BuildCallFlags(execution.Isolation.Interference, execution.Isolation.State),
+		Caller:              rms.NewReference(e.parentObjectReference),
+		Callee:              rms.NewReference(e.class),
 		CallSiteMethod:      e.constructor,
-		CallSequence:        0, // must be filled in the caller
-		KnownCalleeIncoming: reference.Global{},
-		CallOutgoing:        reference.Global{}, // must be filled in the caller
-		Arguments:           e.arguments,
+		KnownCalleeIncoming: rms.NewReference(reference.Global{}),
+		CallOutgoing:        rms.NewReference(reference.Global{}), // must be filled in the caller
```

Comment:
"filled BY the caller"

Reply:
fixed in https://github.com/insolar/assured-ledger/pull/766